src/styles/form.css

```diff
@@ -1 +1,30 @@
 /* form */
+
```

Comment:
To me it sounds like the next iteration. First of all, we need a confirmation from Nellie that this is in fact the case. It seems strange to me that it only changes for focused state.
Also, we need to confirm if can be done in Semantic UI without introducing very specific selectors. We're not sure how FAST is going to use them. There're a few different ways components can be used in Forms (_please take a look at Checkbox PR_). The solution you suggested might apply to this specific scenario, but not others. Once we get a confirmation, this might be another thing to add.

Reply:
And it's a very good catch - thank you! I missed it

Reply:
I think it's easy for us to add, if we decide to. I'd only changed selector a bit:
```
.ui.form .field:not(.error) label:has(+ .ui.input > input:focus) {
  color: var(--primary-pms-362_900);
}
```
Maybe keep it as a comment on a ticket for now?